src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -728,89 +687,83 @@ def __call__(self):
 
             if socket.getfqdn().endswith("cern.ch"):
                 self.handleSpecialCERNMergeSettings(funcName)
-
         else:
+            self.logger.info("DEBUG: Now in the none scenario to load PSET")
             try:
                 self.loadPSet()
             except Exception as ex:
                 self.logger.exception("Error loading PSet:")
                 raise ex
 
+        dummyTweak = PSetTweak()
+        procScript = "edm_pset_tweak.py"
+        dummypsetTweakJson = os.path.join(self.stepSpace.location, "PSetTweak_dummy.json")
```

Comment:
Can you please clarify - through a docstring - what this tweak_dummy object is?

Reply:
Ah! I actually don't need this. This is something I added when I was debugging some issues. I will remove it.